operations/firmware/Update_Firmware_with_FAS.md

```diff
@@ -59,7 +59,8 @@ Table 1. Upgradable Firmware Items
 | Cray             | `chassisBMC` | `BMC`, `Recovery`                                                                                            |
 | Cray             | `routerBMC`  | `BMC`, `Recovery`                                                                                            |
 | Gigabyte         | `nodeBMC`    | `BMC`, `BIOS`                                                                                                |
-| HPE              | `nodeBMC`    | `iLO 5`, `iLO 6`, `System ROM` ,`Redundant System ROM`                                                                |
+| HPE              | `nodeBMC`    | `iLO 5`, `iLO 6`, `System ROM` ,`Redundant System ROM`                                                       |
+| `Foxconn`        | `nodeBMC`    | `bmc_active`, `bios_active`, `erot_active` ,`fpga_active`, `pld_active`                                      |
```

Comment:
Adding ` around Foxconn causes it to render differently than the other manufacturers in the table.  No need for the ` if you add Foxconn to .spelling in the root directory.